org.sixtyninestudios.kublit.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 69 Studios: A Kallex Company -->
+<component type="desktop-application">
+  <id>org.sixtyninestudios.kublit</id>
+  <name>Kublit</name>
+  <summary>69 studios Kublit</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  <launchable type="desktop-id">org.sixtyninestudios.kublit.desktop</launchable>
+  <content_rating type="oars-1.0">
```

Comment:
Please use oars-1.1.

Use this website: https://hughsie.github.io/oars/generate.html